### PR TITLE
dataconnect: .gitignore updated with some files generated by the data connect toolkit and/or firebase-tools

### PR DESCRIPTION
**Pull request by dconeybe**

### CODE REVIEW

File: dataconnect/.gitignore [hunk 1]
```diff
@@ -15,3 +15,7 @@
 local.properties
 .dataconnect/
 .firebaserc
+.firebase/
+dataconnect-debug.log
+firebase-debug.log
+ui-debug.log
```
Comment (thatfiredev) [inline dataconnect/.gitignore:21]: Shall we do `*-debug.log` or even `*.log` ? I can't imagine us adding any `.log` files to the repo. And even if that happens at some point, we could update the .gitignore to allow it.

Comment (dconeybe) [reply dataconnect/.gitignore:21]: Sounds good . I'll make that change today.

Comment (dconeybe) [reply dataconnect/.gitignore]: Done.

**Review by thatfiredev — APPROVED** [review]
LGTM

**Pull request merged**